engine/verification/assigner/engine_test.go

```diff
@@ -0,0 +1 @@
+package assigner
```

Comment:
is this suppose to have tests

Reply:
yes, but not in this PR, but in Yahya's PR. 

This PR's focus is to create the job consumers and feed the job to the engine. 

The implementation of the engine and its tests will be implemented later.